packages/core/src/operator/operator-runtime/listenForChallengesCallback.ts

```diff
@@ -29,7 +29,7 @@ export async function listenForChallengesCallback(challengeNumber: bigint, chall
             });
     }
 
-    operatorState.cachedLogger(`Received new challenge with number: ${challengeNumber}.`);
+    operatorState.cachedLogger(`Received new challenge with number: ${challengeNumber}. Delayed challenges will still accrue rewards.`);
```

Comment:
Im personally not the best person for texts but I feel this should tell something about the rewards being calculated from the time passed between challenges like `Delayed challenges will still accrue the same ratio of rewards based on the time passed.`

We can also request that from Mark, but thats up to you. Not a requested change from my side.